tests/test_views.py

```diff
@@ -24,5 +25,6 @@ def test_get(self) -> None:
         self.assertTemplateUsed(response, 'buza/user_detail.html')
 
         self.assertContains(response, 'Test User', count=2)
+        self.assertContains(response, '<img src="/media/example.jpeg">', count=1)
```

Comment:
I see what you mean by adding the changes and adding the tests for the changes together

Reply:
Cool. :)